### PR TITLE
added teams admin center

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4332

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @scanum, for your information

Thank you

File: Teams/private-channels-life-cycle-management.md [hunk 1]
```diff
@@ -175,7 +175,11 @@ You can try these commands through [Graph Explorer](https://developer.microsoft.
 
 You may want to list out the owners and members of a private channel to decide whether you need to promote certain members of the private channel to an owner. This can happen when you have owners of private channels who have left the organization and the private channel requires admin help to claim ownership of the channel.
 
-As an admin, you can use PowerShell or Graph APIs commands to query these URLs.
+As an admin, you can use Microsoft Teams admin center, PowerShell or Graph API to perform these actions.
```
Comment (JohanFreelancer9) [inline Teams/private-channels-life-cycle-management.md:178]:
```suggestion
As an admin, you can use Microsoft Teams admin center, PowerShell, or Graph API to perform these actions.
```

File: Teams/private-channels-life-cycle-management.md [hunk 2]
```diff
@@ -175,7 +175,11 @@ You can try these commands through [Graph Explorer](https://developer.microsoft.
...
+
+### Using Microsoft Teams admin center
+
+To learn how to manage group members using admin center, see [Manage teams in the Microsoft Teams admin center](https://docs.microsoft.com/microsoftteams/manage-teams-in-modern-portal)
```
Comment (JohanFreelancer9) [inline Teams/private-channels-life-cycle-management.md:182]:
```suggestion
To learn how to manage group members using the admin center, see [Manage teams in the Microsoft Teams admin center](https://docs.microsoft.com/microsoftteams/manage-teams-in-modern-portal).
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Made some minor edits and merged.

**Pull request merged**